styleguide/derek/global-components/modal/_region-select.scss

```diff
@@ -128,3 +128,27 @@
     column-gap: 15px;
   }
 }
+
+@media only screen and (max-width: $screen-xs-min - 1) {
```

Comment:
I don't think we want to stop at 479px here.   Some android devices use 480px and there's still some overlap there: 
![image](https://user-images.githubusercontent.com/5263371/60297271-6fb03200-98ed-11e9-9d42-7a4fb63ccb53.png)

I'd say either set it to `$screen-xs-min` (480px) or `$screen-xs-max` (767px).